SharedPackages/BrowserServicesKit/Sources/Subscription/DeadTokenRecoverer.swift

```diff
@@ -22,10 +22,12 @@ import os.log
 
 public actor DeadTokenRecoverer {
 
-    private static var recoveryAttemptCount: Int = 0
+    private var recoveryAttemptCount: Int = 0
```

Comment:
all var and func are not instance and not static, so they are isolated in the actor